server/src/db/seeders/DogParksSeeder.js

```diff
@@ -0,0 +1,41 @@
+import { DogPark } from "../../models/index.js"
+
+class DogParksSeeder {
+    static async seed() {
+
+        const dogParks = [{
```

Comment:
the open curly bracket should be moved to the next line, so that it is inline with the close curly bracket